### PR TITLE
[WR] Samesite cookie object

### PR DESCRIPTION
**Pull request by kodster28**
Addresses PCX-2037.

### CODE REVIEW

File: products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+However, this behavior can prevent a waiting room from queueing a user properly if that waiting room is embedded in an iFrame. The waiting room depends on the [`__cfwaitingroom` cookie](/reference/waiting-room-cookie) to track a user in the queue. But, since the browser blocks the cookie from reaching the waiting room by default, an active and queueing waiting room cannot queue the user and will never let them access the application.
+
+## Available options
+
+To customize how your waiting room responds to cookies, include the `cookie_attributes` object when you [create a Waiting Room](https://api.cloudflare.com/#waiting-room-create-waiting-room) (only available via the API).
+
+Available options include:
+
+- `samesite`: Configures the `SameSite` attribute on the waiting room cookie:
```
Comment (marciocloudflare) [inline products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md:22]: `samesite` and `SameSite`? Should the second one be lowercase as well? The same question for line 29, `secure` and `Secure`

File: products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+If you are embedding a waiting room in an iFrame, specify the following values on `cookie_attributes` object when [creating a Waiting Room](https://api.cloudflare.com/#waiting-room-create-waiting-room) (only available via the API):
+
+- `samesite`: `none`
+- `secure`: 
+
+    - If you have [**Always Use HTTPS**](https://support.cloudflare.com/hc/articles/204144518#h_a61bfdef-08dd-40f8-8888-7edd8e40d156) enabled, set to `auto`.
+    - If you have **Always Use HTTPS** disabled, set to `always`.
+
+<details>
```
Comment (marciocloudflare) [inline products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md:44]: Are these 2 `details` examples? Should this have a title explaining it?

File: products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+<summary>Request</summary>
+<div>
+
+```json
```
Comment (marciocloudflare) [inline products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md:48]: I think this should be `bash` not `json`? But Pedro is the resident expert :)

File: products/waiting-room/src/content/how-to/create-via-api.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ The following parameters are optional:
 
 The following example API request configures a waiting room.
 
-```shell
+```json
```
Comment (marciocloudflare) [inline products/waiting-room/src/content/how-to/create-via-api.md:46]: same comment here. I think this should be `bash` not `json`

**Review by marciocloudflare — APPROVED** [review]

File: products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md (hunk 1 above)
Comment (kodster28) [reply products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md:22]: Yeah, there's a difference in casing between the actual properties on the cookie and the param options we have in the API.

File: products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md (hunk 3 above)
Comment (kodster28) [reply products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md:48]: Based on his recent comment, it sounds like json would be the better option here... but totally agree that I'm just looking for Pedro's guidance here.

https://chat.google.com/room/AAAA8z-11tc/G6A7rILEss4

File: products/waiting-room/src/content/how-to/create-via-api.md (hunk 4 above)
Comment (kodster28) [reply products/waiting-room/src/content/how-to/create-via-api.md:46]: Same comment as above https://chat.google.com/room/AAAA8z-11tc/G6A7rILEss4

File: products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md (hunk 2 above)
Comment (kodster28) [reply products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md:44]:
```suggestion
### Example

<details>
```

File: products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+- `samesite`: Configures the `SameSite` attribute on the waiting room cookie:
+
+    - **auto** (default): Meant to be as flexible as possible, defaulting to **lax** but becoming **none** if you have enabled [**Always Use HTTPS**](https://support.cloudflare.com/hc/articles/204144518#h_a61bfdef-08dd-40f8-8888-7edd8e40d156).
+    - **lax**: Cookies are not sent on normal cross-site subrequests (for example to load images or frames into a third party site), but are sent when a user is navigating to the origin site
+    - **strict**: Cookies will only be sent in a first-party context.
+    - **none**: Cookies will always be sent.
+
+- `secure`: Configures the `Secure` attribute on the waiting room cookie, which requires the request to be made over `https`:
+
+    - **auto** (default): Meant to be as flexible as possible, defaulting to **always** but becoming **never** if you have enabled [**Always Use HTTPS**](https://support.cloudflare.com/hc/articles/204144518#h_a61bfdef-08dd-40f8-8888-7edd8e40d156).
```
Comment (tc80) [inline products/waiting-room/src/content/additional-options/embed-waiting-room-in-iframe.md:31]: @kodster28 this is *never* by default and is set to *always* on-the-fly if Always Use HTTPS is enabled (it is the other way around)!

**Pull request merged**